docs/DeveloperGuide.adoc

```diff
@@ -345,6 +345,62 @@ image::UndoRedoActivityDiagram.png[width="200"]
 **Cons:** Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things. +
 // end::undoredo[]
 
+// tag::datestoring[]
+=== Date Storing mechanism
```

Comment:
maybe the 's' in 'storing' can be in lowercase for standardization? Minor issue though.